src/sc_crawler/vendors/aliyun.py

```
@@ -0,0 +1,619 @@
+    else:
+        request = DescribeRegionsRequest.DescribeRegionsRequest()
+        response = client.do_action_with_exception(request)
+        region_info_in_list = json.loads(response.decode("utf-8")) 
+    return region_info_in_list
+
+def get_zones(tempcl):
+    """
+    Gathers the zones available in given region. Takes a client object as a parameter. Returns a dictionary with zone info. 
+    """
+    #client = get_client()
+    region_list = get_regions(one_region_query=ONE_REGION_QUERY)
```

Ruff (0.14.5)

Local variable `region_list` is assigned to but never used

Remove assignment to unused variable `region_list`

(F841)